lib/analysis.js

```diff
@@ -61,6 +61,14 @@ function normalize( input ){
   .map( function( synonym ){
     return removeAccents( lowercase( synonym ) );
   })
+  // generic synonym contractions
+  .map( function( synonym ){
+    return synonym
```

Comment:
This will analyze `Gressoney-Saint-Jean` to `gressoneysaintjean` and `gressoney st jean`.  Is this what you intended?